src/types/UserGroupChangemakerPermission.ts

```diff
@@ -0,0 +1,41 @@
+import { ajv } from '../ajv';
+import { Permission } from './Permission';
+import type { JSONSchemaType } from 'ajv';
+import type { Writable } from './Writable';
+import type { KeycloakId } from './KeycloakId';
+
+interface UserGroupChangemakerPermission {
+	readonly keycloakOrganizationId: KeycloakId;
+	readonly permission: Permission;
```

Comment:
While order is technically not significant here, for consistency this should be in the Latin/Cherokee/Yoda order of subject-object-verb.